notes.org

```diff
@@ -37,24 +37,29 @@
 		 let posts_in_blog = query!(Post, &conn, blog = {blog}).load(&conn)
 	 }
 	 #+END_SRC
+
 	 struct Posts
+
+	 #+BEGIN_SRC sql
 	 CREATE TABLE posts (
-  id SERIAL PRIMARY KEY,
-  title VARCHAR NOT NULL,
-  body TEXT NOT NULL,
-  published BOOLEAN NOT NULL DEFAULT 'f'
-)
+		id SERIAL PRIMARY KEY,
+		title VARCHAR NOT NULL,
+		body TEXT NOT NULL,
+		published BOOLEAN NOT NULL DEFAULT 'f'
+	 )
+	 #+END_SRC
+
 * Roadmap
 ** vNext
-	 + foreign key constraints in db, cascade setting
+	 + foreign key constraints in db, cascade setting [[https://github.com/Electron100/butane/issues/21]]
 	 + incremental save
 	 + Many backreferences
```

Comment:
I guess this is simply a function to take the caller from a `Many` instance to the instance that "(owner table, key)" points to?
Seems so, and seems not too hard to do, and also quite useful.

Reply:
Correct